networkit/cpp/embedding/test/FiniteEmbeddingTest.cpp

```diff
@@ -50,19 +57,16 @@ TEST_F(FiniteEmbeddingTest, testNode2VecOnGraphWithNonContinuousIds) {
     G.addEdge(0, 1);
     G.addEdge(1, 2);
     G.removeNode(1);
-    EXPECT_THROW(NetworKit::Node2Vec{G}, std::runtime_error);
+    EXPECT_THROW(Node2Vec{G}, std::runtime_error);
 }
 
 TEST_F(FiniteEmbeddingTest, testFiniteEmbeddingOnSmallGraph) {
 
-    NetworKit::METISGraphReader reader;
-
-    NetworKit::Graph graph = reader.read("input/karate.graph");
-
-    auto algo = NetworKit::Node2Vec(graph, .3, 1.3);
+    METISGraphReader reader;
+    Graph graph = reader.read("input/karate.graph");
 
+    auto algo = Node2Vec(graph, .3, 1.3);
```

Comment:
I think this is simpler:
```suggestion
    Node2Vec algo(graph, .3, 1.3);
```